lib/dialects/abstract/query-generator.js

```diff
@@ -1519,8 +1535,26 @@ const QueryGenerator = {
 
         mainQueryOrder.push(this.quote(t, model, false, '->'));
       }
+    } else if (typeof options.order === 'string') {
+      // split
+      for (let order of options.order.split(',')) {
+        // get options
+        const orderOption = getOrderOption(order);
+        if(orderOption.index > -1){
+          order = order.substring(0, orderOption.index);
+        }
+        order = _.trim(order);
+        let sql = this.quote(order, model, false, '->');
+        if(orderOption.option){
+          sql += ' ' + orderOption.option;
+        }
+        if (subQuery) {
+          subQueryOrder.push(sql);
+        }
+        mainQueryOrder.push(sql);
+      }
```

Comment:
We don't support raw strings in other places in favor of explicit Sequelize.literal(), like `attributes` etc. `order` is one of the last places where this is still supported. Sequelize codebase is already very complicated, there is also the desire to keep complexity low. Can you really guarantee that your algorithm can't be spoofed? Can you guarantee you handle every possible ORDER syntax? Just look at the possibilities for Postgres: https://www.postgresql.org/docs/9.1/static/queries-order.html, these also can vary through dialects. Since we don't test all the edge cases here there is still the potential to break someone's code and I don't want to commit to parsing every case here just to then put it inside the SQL again. Essentially this would introduce a mini DSL for `order` that is probably a subset of what's possible with SQL and it would still be an attack vector for SQL injection. I would rather target v4 and tell people to wrap in Sequelize.literal().